smart-contracts/tasks/deploy/bridge-proxy.ts

```diff
@@ -81,6 +84,39 @@ task('deploy:bridge-proxy', 'Deploy a bridge proxy').setAction(
       console.log(
         `ArbOrbit bridge deployed at: ${await proxyBridge.getAddress()}`
       )
+    } else if (type === 'zksync') {
+      let zkSyncBridgeAddress: string
+      const l2SharedDefaultBridge = bridges.zksync!.l2SharedDefaultBridge!
+      const l1SharedDefaultBridge = bridges.zksync!.l1SharedDefaultBridge!
+      if (network.zksync) {
```

Comment:
Le's not use the name here, but rathe a flag to check if this is a L1 (or not an L2). I can imagine we would wnat to deploy to other zksync L2.

Reply:
this is an official flag for the zksync lib that works for all L2 (included zksync sepolia). It is used by the solc compiler and specified in the `networks` object of hardhat

Reply:
ohh! ok I thought `network` was coming from our `networks` package. Can we use it instead of `hre.network`. The flag will be useful in other contexts.